diff --git a/node_modules/@react-native-community/cli-platform-android/native_modules.gradle b/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
index 30ddaf1..598f36e 100644
--- a/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
+++ b/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
@@ -11,6 +11,7 @@ import android.content.Context;
 import android.content.res.Resources;
 
 import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainPackageConfig;
 import com.facebook.react.shell.MainReactPackage;
 import java.util.Arrays;
 import java.util.ArrayList;
@@ -20,13 +21,25 @@ import java.util.ArrayList;
 public class PackageList {
   private Application application;
   private ReactNativeHost reactNativeHost;
+  private MainPackageConfig mConfig;
+
   public PackageList(ReactNativeHost reactNativeHost) {
-    this.reactNativeHost = reactNativeHost;
+    this(reactNativeHost, null);
   }
 
   public PackageList(Application application) {
+    this(application, null);
+  }
+
+  public PackageList(ReactNativeHost reactNativeHost, MainPackageConfig config) {
+    this.reactNativeHost = reactNativeHost;
+    mConfig = config;
+  }
+
+  public PackageList(Application application, MainPackageConfig config) {
     this.reactNativeHost = null;
     this.application = application;
+    mConfig = config;
   }
 
   private ReactNativeHost getReactNativeHost() {
@@ -48,7 +61,7 @@ public class PackageList {
 
   public ArrayList<ReactPackage> getPackages() {
     return new ArrayList<>(Arrays.<ReactPackage>asList(
-      new MainReactPackage(){{ packageClassInstances }}
+      new MainReactPackage(mConfig){{ packageClassInstances }}
     ));
   }
 }
